### PR TITLE
Disable notification emails 

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/8904

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-runtime/src/pages/account.vue [hunk 1]
```diff
@@ -241,24 +266,143 @@ export default defineComponent({
...
+      }
+    }
+
+    const updateDisableEmailNotifications = async (option) => {
+      option = !option
+      console.log(option)
+      const bundle = loadBundlesList.last?.value
+
+      const settingsId = unref(valuesList).find(
+        (cV) => cV.identifier.setting === "'disable email notifications"
```
Comment (dschmidt) [inline packages/web-runtime/src/pages/account.vue:328]: Is the `'` there on purpose?
If yes, can you add a comment why it's there? It *looks* wrong

**Review by dschmidt — COMMENTED** [review]
There's quite a bit code duplication for the value saving, could you reduce that?

I assume something like 

```
  const updateDisableEmailNotifications = async (option) => {
    await saveValue({
      identifier: 'disable email notifications',
      value: option,
      errorMessage: $gettext('Unable to save notifications email setting…')
    })
    disableEmailNotificationsValue.value = option
  }
```

should be all we need for a specific value.

File: packages/web-runtime/src/pages/account.vue (hunk 1 above)
Comment (JanAckermann) [reply packages/web-runtime/src/pages/account.vue]: no + not ready to review =)

File: packages/web-runtime/src/pages/account.vue [hunk 2]
```diff
@@ -100,45 +104,60 @@
           <gdpr-export />
         </dd>
       </div>
+      <div class="account-page-notification oc-mb oc-width-1-2@s">
```
Comment (JanAckermann) [inline packages/web-runtime/src/pages/account.vue:107]: IsSettingsServiceSupported condition missing

**Comment by JanAckermann** [comment]
TBD: 
- [ ] Add unit tests

**Comment by ownclouders** [comment]
Results for <strong>acceptance oCIS</strong> https://drone.owncloud.com/owncloud/web/35211/69/1
:boom: The oCISTrashbinJourney tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>acceptance oC10</strong> https://drone.owncloud.com/owncloud/web/35211/70/1
:boom: The oC10IntegrationApp1 tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>acceptance oCIS</strong> https://drone.owncloud.com/owncloud/web/35211/66/1
:boom: The oCISSharingPublic2 tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>acceptance oCIS</strong> https://drone.owncloud.com/owncloud/web/35211/68/1
:boom: The oCISUploadMove tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>acceptance oC10</strong> https://drone.owncloud.com/owncloud/web/35211/71/1
:boom: The oC10IntegrationApp2 tests pipeline failed. The build has been cancelled.

File: packages/web-runtime/src/pages/account.vue [hunk 3]
```diff
@@ -152,113 +171,202 @@ export default defineComponent({
...
+          (cV) => cV.identifier.setting === 'language'
+        )
+        selectedLanguageValue.value = languageConfiguration
+          ? unref(languageOptions)?.find(
+              (lO) => lO.value === languageConfiguration.value?.listValue?.values?.[0]?.stringValue
+            )
+          : unref(languageOptions)?.find((o) => o.default)
+
+        const disableEmailNotificationsConfiguration = unref(valuesList)?.find(
+          (cV) => cV.identifier.setting === 'disable email notifications'
```
Comment (kulmann) [inline packages/web-runtime/src/pages/account.vue:346]: I hope that my wish to change this to `disable-email-notifications` will be respected by the backend team 🙈

**Review by kulmann — APPROVED** [review]
Love it! 💪

**Pull request merged**